### ISSUE TITLE
MagicString.indent is slow

### ISSUE THREAD

**Issue by Victorystick** [Rich-Harris/magic-string#94]
According to https://github.com/rollup/rollup/issues/723#issuecomment-235984088, MagicString.indent can take up quite a bit of Rollup's bundle time. It may be worthwhile to try to optimize it.

**Comment by Rich-Harris** [comment]
Yeah. It's not any slower than any other operation, it's just that it needs to do a lot more of it (Rollup only needs to make small edits here and there until `indent()`, at which point every line is affected) – would definitely be good to make it faster, but it may be the case that we're better off just disabling indentation by default. Won't be able to take a look any time soon (am on a harsh deadline hence my recent radio silence!) but keen to improve this.
